packages/medusa/src/models/order-sales-channel.ts

```diff
@@ -0,0 +1,27 @@
+import { BeforeInsert, Column, Index } from "typeorm"
+import { MedusaV2Flag } from "@medusajs/utils"
+
+import { generateEntityId } from "../utils"
+import { SoftDeletableEntity } from "../interfaces"
+import { FeatureFlagEntity } from "../utils/feature-flag-decorators"
+
+@FeatureFlagEntity([MedusaV2Flag.key, "sales_channels"])
+export class OrderSalesChannel extends SoftDeletableEntity {
```

Comment:
thought: `CartSalesChannel` contains an ID, but this one doesn't. Even though it has an `beforeInsert` setting the `id`. I think we should be consistent across the models

Reply:
It has an id from BaseEntity, but in the case of `CartSalesChannel`, id is overridden since it's not a primary key. But I agree we should follow the same approach here as with the cart entity. I will update the model.